### DISCUSSION TITLE
what is best way to use  multiple  subgraph ?

### DISCUSSION THREAD

**Discussion by baixiaoyu2997** [graphprotocol/graph-tooling#1000] [Q&A] [answered]
I'm not sure if this is possible，  just to list possible problems：
1. deploy multiple subgraph to same graph-node?
    1. if i do this,  should i split subgraphs to  multiple project? or in one project?

**Comment by azf20** [comment, marked as answer]
Hi! You can indeed deploy multiple subgraphs to the same graph-node. We run 1-2k subgraphs on a single instance in some cases

**Reply by baixiaoyu2997** [reply]
thanks，yesterday， i try with that, it success, but still shocked by this number,  1-2k，omg 😱

**Reply by baixiaoyu2997** [reply]
@azf20  sorry to bother you again， i only use two subgraph on one `graph-node`, but it has used 65g of hard drive space, is this normal?  how can i optimize it

**Reply by azf20** [reply]
hi @baixiaoyu2997, it depends a lot on the specific subgraph, if it is saving lots of entities then that seems possible.
